### ISSUE TITLE
[Request]: Add simple map to the Web API Explorer

### ISSUE THREAD

**Issue by eneemann** [agrc/api.mapserv.utah.gov#172]
I frequently use the [Web API Explorer](https://api.mapserv.utah.gov/) to check an address or test fixes to a bad address.  When I get the results and coordinates, what I really want is to plot the location and visually verify that the result is where I expect it to be or verify that it makes sense.

### **Proposed Solution:**
It would be a huge help and time-saver if a simple embedded webmap was right below the API results, so users could visually see the geocoded location plotted on a basemap (Utah Hybrid?).

**Comment by nathankota** [comment]
cool enhancement idea

**Comment by steveoh** [comment]
Yeah it is a great idea. I'm thinking to do something like open result in atlas or something and pipe people over there rather than embedding it maybe?

**Comment by gregbunce** [comment]
i support the idea of enhancing atlas.  it would be nice if it showed you the match score and layer that was matched. like the API. maybe a sidebar panel that you can expand or something that you have to open so it's not in the way.

**Comment by steveoh** [comment]
Atlas was enhanced.
